src/main/java/randoop/util/WeightedList.java

```diff
@@ -2,19 +2,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import randoop.BugInRandoopException;
 
 public class WeightedList {
 
   private List<WeightedElement> theList;
+  private List<Double> cumulativeWeights;
```

Comment:
Changes removed (used only in early versions of our project's implementation).  Thanks!